sw/device/lib/dif/dif_sram_ctrl.h

```diff
@@ -11,256 +11,276 @@
  * Functions
  */
 
-/**
- * The SRAM Controller can perform destructive operations on data in memory.
- * These operations invalidate the stack and other data that the C runtime
- * requires. Such functionality has been intentionally left out of this library
- * and will need to be triggered by assembly code that does not require a valid
- * stack or initialized static data. Such functionality includes:
- *
- * - Requesting a new scrambling key.
- * - Initializing the contents of memory.
- *
- * For the same reason there is no accessor function in this library for the
- * `status.ESCALATED` bit. If that bit is set then memory is inaccessible.
- */
-
-#include <stdbool.h>
 #include <stdint.h>
 
-#include "sw/device/lib/base/macros.h"
-#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/autogen/dif_sram_ctrl_autogen.inc"
```

Comment:
Should be including the .h instead of the .inc, since the .inc will be deleted as @sriyerg mentioned above.